recipes-core/tdx-enc-handler/tdx-enc-handler_1.0.bb

```diff
@@ -0,0 +1,31 @@
+SUMMARY = "Scripts to handle encryption on Toradex modules"
+
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+SRC_URI = "\
+    file://tdx-enc-cleartext.sh \
+    file://tdx-enc-handler.service \
+"
+
+RDEPENDS:${PN} = "\
+    openssl-bin \
+    cryptsetup \
+    e2fsprogs-mke2fs \
+"
+
+inherit systemd
+
+SYSTEMD_SERVICE:${PN} = "tdx-enc-handler.service"
+
+do_install() {
+    install -d ${D}${sbindir}
+    install -m 0755 ${WORKDIR}/tdx-enc-${TDX_ENC_KEY_BACKEND}.sh ${D}${sbindir}/tdx-enc.sh
+
+    sed -i 's|###TDX_ENC_KEY_FILE###|${TDX_ENC_KEY_FILE}|g' ${D}${sbindir}/tdx-enc.sh
+    sed -i 's|###TDX_ENC_STORAGE_LOCATION###|${TDX_ENC_STORAGE_LOCATION}|g' ${D}${sbindir}/tdx-enc.sh
+    sed -i 's|###TDX_ENC_STORAGE_MOUNTPOINT###|${TDX_ENC_STORAGE_MOUNTPOINT}|g' ${D}${sbindir}/tdx-enc.sh
```

Comment:
Just to follow a pattern in this layer, how about using `@@<variable-to-be-replaced>@@` here? See https://github.com/toradex/meta-toradex-security/blob/kirkstone-6.x.y/recipes-bsp/imx-mkimage/files/mx8_sign.sh#L109.

Reply:
Oh yeah, I forgot about this. I will follow the same patter then. Thanks for pointing that out.